test/upgrade/service_postupgrade_test.go

```diff
@@ -62,6 +74,30 @@ func TestBYORevisionPostUpgrade(t *testing.T) {
 	}
 }
 
+func configUpdated(t *testing.T, clients *test.Clients, serviceName string) bool {
+	configObj, err := clients.ServingAlphaClient.Configs.Get(serviceName, metav1.GetOptions{})
+	if err != nil {
+		t.Fatalf("Failed to get Configuration: %v", err)
+	}
+	if configObj.Generation != 1 && configObj.ObjectMeta.Generation != 1 {
```

Comment:
These are the same fields. ObjectMeta is an anonymous field in the struct. You should only need to compare one.